tracim/development.ini.base

```diff
@@ -235,6 +237,16 @@ email.reply.use_txt_parsing = true
 # it's just an empty file use to prevent concurrent access to imap unseen mail
 email.reply.lockfile_path = %(here)s/email_fetcher.lock
 
+## Jitsi_meet_integration
+jitsi_meet.activated = False
+jitsi_meet.domain = your_jitsi_meet_instance
```

Comment:
domain can be a domain or an ip address ? or only a domain ? You should probably add a comment and put in the comment an example. Eg 127.0.0.1 or meet.jitsi.algoo.fr as an example ?